### PR TITLE
Etch API Support for Embedded Signers

### PR DESCRIPTION
**Pull request by Winggo**
## Description of the change
Goal - achieve this flow for embedded etch signatures:
1) They create a packet with some signers that have signerType: 'embedded'
2) They call the generateEtchSignURL({signerEid, clientUserId}) mutation to get a sign URL for the signer
3) The user signs
4) Instead of our finish page, we redirect to a page in their system

Changes made:
- create-etch-packet, generate-etch-sign-url node-anvil API and example scripts work well
- anvil-node docs updated

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Development

- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development
- [ ] No previous tests unrelated to the changed code fail in development

### Code review

- [ ] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached.
- [ ] At least one reviewer has been requested
- [ ] Changes have been reviewed by at least one other engineer
- [ ] The relevant project board has been selected in Projects to auto-link to this pull request

### CODE REVIEW

**Review by benogle — APPROVED** [review]

**Comment by newhouse** [comment]
@Winggo Now that #16 is in it's time to rebase your branch and let's have a lookie!

File: example/script/generate-etch-sign-url.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  }
+
+  const client = new Anvil(clientOptions)
+
+  const variables = {
+    signerEid,
+    clientUserId,
+  }
+
+  const { statusCode, data, errors } = await client.generateEtchSignUrl({ variables })
```
Comment (newhouse) [inline example/script/generate-etch-sign-url.js:20]: Very minor thing, but seems a little funny to me to have the URL returned be in a thing called `data`...perhaps the `generateEtchSignUrl` method could/should massage the response to rename it to `url`?  Wonder what @benogle thinks...

E.g.
```js
async generateEtchSignUrl(...) {
  const { data, ...rest } = await this.requestGraphQL(...)
  return {
    url: data,
    ...rest
  }
}
```

**Review by newhouse — COMMENTED** [review]
1 comment on the code, but also I think that the `README` should be updated to include documentation for the `generateEtchSignUrl` method (params, response ,etc)...or I can add it myself sometime between now and when we release these changes as `v2.2` or whatever...

**Comment by Winggo** [comment]
> 1 comment on the code, but also I think that the `README` should be updated to include documentation for the `generateEtchSignUrl` method (params, response ,etc)...or I can add it myself sometime between now and when we release these changes as `v2.2` or whatever...

Didn't see this earlier, gonna be adding this now

File: example/script/generate-etch-sign-url.js (hunk 1 above)
Comment (Winggo) [reply example/script/generate-etch-sign-url.js:20]: I think this would be better

**Review by newhouse — APPROVED** [review]
ya boyeeeee

**Pull request merged**